src/components/DeleteKebab/DeleteKebab.tsx

```diff
@@ -35,6 +35,10 @@ const DeleteKebab = ({
     onFocus();
   };
 
+  const triggerSnapshot = () => {
+    console.log('Snapshot triggered!');
```

Comment:
This is a console log, it does not call the API. You need to use the new API that was merged recently, adding it to the front-end. 
[Here](https://github.com/content-services/content-sources-backend) is the backend PR to git checkout.